core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -218,7 +219,8 @@ object TopicCommand extends Logging {
                 print("\tPartition: " + partitionId)
                 print("\tLeader: " + (if(leader.isDefined) leader.get else "none"))
                 print("\tReplicas: " + assignedReplicas.mkString(","))
-                println("\tIsr: " + inSyncReplicas.mkString(","))
+                print("\tIsr: " + inSyncReplicas.mkString(","))
+                if (!describeConfigs) println("\t" + markedForDeletion) else println()
```

Comment:
Right, considering it's minor, can we simply update the KIP and send an update to the mailing list ?